app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -0,0 +1,114 @@
+package ch.hikemate.app.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.KeyboardArrowRight
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import ch.hikemate.app.R
+import ch.hikemate.app.ui.map.MapScreen
+
+@Composable
+fun HikeCard(
+    title: String,
+    altitudeDifference: Int,
```

Comment:
I agree with you, there might be attributes that would make more sense here. Note that the `HikeCard` component already existed, I just moved it from `MapScreen.kt` to its own file, so I kept it pretty much as it was, just made a few adjustments.

I believe we put the altitude difference here because the `HikeCard` was supposed to display the altitude profile (the graph) too, so it made sense to have the total altitude difference on the side. But we might want to change it later and put something such as the length of the hike.

This would probably be another task for another PR though. Do you want to create a task for this in the backlog? I'm not sure what you would put in there, that's why I don't do it right now myself.

Reply:
Opened a discussion on telegram

Reply:
As this is going to be another task anyway, I'm marking the conversation as resolved.